testcases/kernel/syscalls/getrandom/Makefile

```diff
@@ -0,0 +1,23 @@
+#
+#  Copyright (c) International Business Machines  Corp., 2001
+#
```

Comment:
Please fix the copyright lines, this one should be something as:

Copyright (C) 2015 Cedric Hnyda <your_email@provider>

And the rest of the files should have this instead of SGI copyright as well.